### PR TITLE
Add Anaconda status

### PR DESCRIPTION
**Pull request by jakirkham**
Please edit the `site/template.html` file w/ your changes.

### CODE REVIEW

**Comment by jakirkham** [comment]
cc @beckermr

File: site/template.html [hunk 1]
```diff
@@ -201,6 +201,7 @@ <h4 id="cloud_services"><a href="#cloud_services" style="color: inherit;">Cloud
         <!-- <div id="ciStatusDiv-circle"><h5><a href="https://status.circleci.com">CircleCI status</a>: No Status Available</h5></div>
         <div id="ciStatusDiv-appveyor"><h5><a href="https://status.appveyor.com">AppVeyor status</a>: No Status Available</h5></div> -->
         <div id="ciStatusDiv-quay"><h5><a href="https://status.quay.io/">Quay.io status</a>: No Status Available</h5></div>
+        <div id="ciStatusDiv-quay"><h5><a href="https://anaconda.statuspage.io">Anaconda status</a>: No Status Available</h5></div>
```
Comment (chrisburr) [inline site/template.html:204]:
```suggestion
        <div id="ciStatusDiv-anaconda"><h5><a href="https://anaconda.statuspage.io">Anaconda status</a>: No Status Available</h5></div>
```

Comment (jakirkham) [reply site/template.html]: Oops 😬 Thanks Chris! 🙏

**Comment by jakirkham** [comment]
Thanks Chris! 😄

Wondering if we need something else here or does it just take time to pick up the status?

Screen Shot 2022-04-26 at 8 48 36 PM

**Comment by jakirkham** [comment]
Interesting, appears to work correctly in Firefox 🤔

Screen Shot 2022-04-26 at 8 59 33 PM

**Comment by jakirkham** [comment]
Also works in a private Safari window, but attempting to bust the browser cache doesn't appear to fix it

**Comment by jakirkham** [comment]
Finally busted the cache. Now it works! 🎉

Screen Shot 2022-04-26 at 9 06 01 PM

**Pull request merged**